tox.ini

```diff
@@ -16,9 +16,10 @@ all_path = {[vars]src_path} {[vars]tests_path}
 [testenv]
 allowlist_externals =
     /bin/bash
-    poetry
+    uv
 set_env =
     PYTHONPATH = {tox_root}/lib:{[vars]src_path}
+    UV_PROJECT_ENVIRONMENT={work_dir}/{env_name} 
```

Comment:
uv chose to not treat `VIRTUAL_ENV` as the active environment. This var setup force uv to respect the "tox way" of creating a different venv for each command.
Without it, `.venv` would be overwritten on each tox invocation